### PR TITLE
feat(hits): introduce hits fullpath in search results

### PR DESCRIPTION
**Pull request by jerrytohvan**
Creating PR for proposal added here: https://github.com/algolia/algoliasearch-zendesk/issues/182#issue-1971967937

### CODE REVIEW

**Comment by jerrytohvan** [comment]
Head's up that I need help and guidance to check if this is the best implementation. I was also encountering blockers from running local environment to test this properly.

**Comment by Jerska** [comment]
This looks like a great addtion! Can you share which blockers you faced on the local setup?

**Comment by jerrytohvan** [comment]
image

@Jerska , i received the above error when running `npm start` and didnt manage to get clean `npm install`. Running node on version 10.24.1 currently.

**Comment by jerrytohvan** [comment]
Hi @Jerska , following up on this if you are able to assist with pull request? :)

**Comment by Jerska** [comment]
Sorry for not getting back to you earlier!

I just merged a PR that should make it easier to run locally, I'd recommend you to pull master again & retry `npm install && npm run dev` (sorry this repo is quite old, and libs that were cutting edge back them are so old today...).

That being said, I can confirm it works:
image

It's a bit heavy in terms of effects for the general library, I have a few changes to recommend that would make them look like so:
image

File: app/css/index.scss [hunk 1]
```diff
@@ -283,6 +283,14 @@
     float: right;
     font-size: 0.8em;
   }
+  .search-result-path {
```
Comment (Jerska) [inline app/css/index.scss:286]: There are trailing whitespaces here:
```suggestion
  .search-result-path {
```

File: app/css/index.scss [hunk 2]
```diff
@@ -283,6 +283,14 @@
     float: right;
     font-size: 0.8em;
   }
+  .search-result-path {    
+    font-size: 0.7em;
+    font-style: italic;
+    color: gray;
+    text-decoration-line: underline;
+    font-weight: bold;
```
Comment (Jerska) [inline app/css/index.scss:291]:
```suggestion
    font-size: 0.9em;
    font-style: italic;
    color: #999;
```

**Review by Jerska — COMMENTED** [review]
Except for a few cosmetic changes, this looks great to me, thank you for the Pull Request!
@sbellone WDYT?

File: app/css/index.scss (hunk 2 above)
Comment (jerrytohvan) [reply app/css/index.scss:291]: Will leave you all with the final decision on styling! :)

**Comment by jerrytohvan** [comment]
Thanks for the review and I've pushed your suggestions @Jerska !

**Comment by jerrytohvan** [comment]
Thank you team @sbellone @Jerska ! 👏
Are there timeline on next version being published for usage? :)

**Comment by Jerska** [comment]
I'll release. :)

**Comment by Jerska** [comment]
Thank you for the contribution!

**Comment by Jerska** [comment]
Had to fight the dependencies of all systems, but it's freshly released in `2.32.0`!
See:
- https://community.algolia.com/zendesk/documentation/#available-options
- https://www.npmjs.com/package/algoliasearch.zendesk-hc?activeTab=versions

**Comment by jerrytohvan** [comment]
thank you all!

**Pull request merged**